### DISCUSSION TITLE
how can I write prompts that will query chatgpt while referencing my doc

### DISCUSSION THREAD

**Discussion by till-tech-do-us-part** [FlowiseAI/Flowise#262] [Q&A]
Hello All,

Would be grateful if someone can throw me a life jacket by answering this for me.

I created my first chatflow...sweet!  

1. Uploading DOCX doc
2. Split into chuncks
3. Embeded
4. Stored in vector db
5. Chained to OpenAI for Q&A

Currently, the Flowise Chat bot interacts with the DOCX doc perfectly.

Questions is...how can I write prompts that will query chatgpt while referencing my DOCX doc?

**Comment by HenryHengZJ** [comment]
hey try to use ConversationalRetrievalQAChain and specify the prompt.

smtg like:
```
I want you to act as a document that I am having a conversation with. Your name is "AI Assistant". You will provide me with answers from the given info. If the answer is not included, say exactly "Hmm, I am not sure." and stop after that. Refuse to answer any question not about the info. Never break character.
```
